<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1547449399466' stop='1547449424419'>
    <name>Navigate to RMT URL</name>
    <title>Navigate to RMT URL</title>
    <test-cases>
        <test-case start='1547449399469' status='passed' stop='1547449421642'>
            <name>Login as HR</name>
            <title>Login as HR</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='e0d57676-0e9b-4fbe-9f52-c98cac26c129-attachment.png' type='image/png' size='85416'/>
            </attachments>
        </test-case>
        <test-case start='1547449421642' status='failed' stop='1547449424418'>
            <name>Check for Activelist records</name>
            <title>Check for Activelist records</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='aa12a215-c6e7-4ab6-9b0b-9feb3c90fc8a-attachment.png' type='image/png' size='54361'/>
            </attachments>
            <failure>
                <message>Failed: unknown error: Element &lt;span class="ng-tns-c14-8 ng-star-inserted">...&lt;/span> is not clickable at point (570, 179). Other element would receive the click: &lt;div _ngcontent-c13="" class="load ng-star-inserted">...&lt;/div>
                  (Session info: chrome=71.0.3578.98)
                  (Driver info: chromedriver=2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac),platform=Linux 4.15.0-43-generic x86_64)</message>
                <stack-trace>WebDriverError: unknown error: Element &lt;span class="ng-tns-c14-8 ng-star-inserted">...&lt;/span> is not clickable at point (570, 179). Other element would receive the click: &lt;div _ngcontent-c13="" class="load ng-star-inserted">...&lt;/div>
                  (Session info: chrome=71.0.3578.98)
                  (Driver info: chromedriver=2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac),platform=Linux 4.15.0-43-generic x86_64)
                    at Object.checkLegacyResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: WebElement.click()
                    at Driver.schedule (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at WebElement.schedule_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
                    at WebElement.click (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2092:17)
                    at actionFn (/usr/local/lib/node_modules/protractor/built/element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at actionResults.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:461:65)
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at Object.clickWhenClickable (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/node_modules/protractor-helper/index.js:90:15)
                    at UserContext.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/SanityTestcases/Activelist.js:22:25)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
                    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
                    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
                    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                From: Task: Run it("Check for Activelist records") in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/SanityTestcases/Activelist.js:18:4)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/SanityTestcases/Activelist.js:7:1)
                    at Module._compile (internal/modules/cjs/loader.js:689:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
                    at Module.load (internal/modules/cjs/loader.js:599:32)
                    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>